src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/ProtectionPolicy/SetAzureRmRecoveryServicesBackupProtectionPolicy.cs

```diff
@@ -29,7 +29,7 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
     /// Update existing protection policy in the recovery services vault
     /// </summary>
     [Cmdlet(VerbsCommon.Set, "AzureRmRecoveryServicesBackupProtectionPolicy",
-        SupportsShouldProcess = true), OutputType(typeof(List<JobBase>))]
+        SupportsShouldProcess = true), OutputType(typeof(JobBase))]
```

Comment:
@twitchax same comment about the [output in the markdown](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/help/Set-AzureRmRecoveryServicesBackupProtectionPolicy.md#outputs) for this cmdlet not being updated